overlay/usr/local/bin/trac-initproject

```diff
@@ -16,7 +16,8 @@ init_trac_project() {
     mv $PROJ_LIB/conf/trac.ini $INI
     ln -s $INI $PROJ_LIB/conf/trac.ini
     chown www-data:www-data $INI
-
+    chmod 0755 $INI
```

Comment:
You might want to use `750` since the file contains a database password when using MySQL or PostgreSQL.

Reply:
My bad.Thank you so much for the revision and the correction.

Reply:
Unfortunately, we don't even have good guidance on proper file permissions in TracInstall: https://trac.edgewall.org/wiki/TracInstall. I'm going to look into updating it.